modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbarBounds.ts

```diff
@@ -7,16 +7,17 @@
 
 import { Bounds, Boxes } from '@ephox/alloy';
 import { InlineContent } from '@ephox/bridge';
-import { Optional } from '@ephox/katamari';
+import { Num, Optional } from '@ephox/katamari';
 import { Scroll, SelectorFind, SugarBody, SugarElement, SugarNode, Traverse, WindowVisualViewport } from '@ephox/sugar';
 
 import Editor from 'tinymce/core/api/Editor';
 
 import * as Settings from '../../api/Settings';
 import { UiFactoryBackstageShared } from '../../backstage/Backstage';
 
-const isVerticalOverlap = (a: Bounds, b: Bounds, threshold: number = 0): boolean =>
-  b.bottom - a.y >= threshold && a.bottom - b.y >= threshold;
+// Note: We want to avoid including with small difference such as 0.001px
+const isVerticalOverlap = (a: Bounds, b: Bounds, threshold: number = 0, precision: number = 2): boolean =>
+  Num.toPrecision(b.bottom - a.y, precision) >= threshold && Num.toPrecision(a.bottom - b.y, precision) >= threshold;
```

Comment:
Why not just make the threshold 0.01? How is this rounding method different?

I worry that there will be weird edge cases with Num.toPrecision given that we are dealing with floating point numbers which are notorious for not being able to exactly represent some decimal numbers.

Reply:
That's a very good point, thanks! I've changed it and removed the `toPrecision` function/tests.